tests/system-broker/tests/testkit/config.go

```diff
@@ -0,0 +1,32 @@
+package testkit
```

Comment:
Ditch the `testkit` directory and have a top-level `tests/system-broker/pkg` directory as this would be the way to go with the new e2e test structure. `testkit` is an insidious and confusing package based on the legacy way of how the e2e tests were written.